### DISCUSSION TITLE
Cannot find module "vue/server-renderer/index.mjs"

### DISCUSSION THREAD

**Discussion by stanpodoba** [nuxt/nuxt#28877] [General]
I can't run an absolutely brand new app. Should I send a bug? Or I do something wrong?

**Reproduction**

`npx nuxi@latest init my-app`
`cd ~/my-app`
`npm install`
`npm run build`
`node .output/server/index.mjs`

then open `http://127.0.0.1:3000/` in browser and get error:

```log
[nuxt] [request error] [unhandled] [500] Cannot find module '/Users/stanpodoba/my-app/.output/server/node_modules/vue/server-renderer/index.mjs' imported from /Users/stanpodoba/my-app/.output/server/chunks/handlers/renderer.mjs
  at new NodeError (node:internal/errors:405:5)  
  at finalizeResolution (node:internal/modules/esm/resolve:327:11)  
  at moduleResolve (node:internal/modules/esm/resolve:946:10)  
  at defaultResolve (node:internal/modules/esm/resolve:1132:11)  
  at nextResolve (node:internal/modules/esm/loader:163:28)  
  at ESMLoader.resolve (node:internal/modules/esm/loader:835:30)  
  at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)  
  at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)  
  at link (node:internal/modules/esm/module_job:76:36)
```

------------------------------
- Operating System: Darwin
- Node Version:     v18.18.1
- Nuxt Version:     3.8.1
- CLI Version:      3.13.1
- Nitro Version:    2.7.2
- Package Manager:  npm@9.8.1
- Builder:          -
- User Config:      compatibilityDate, devtools
- Runtime Modules:  -
- Build Modules:    -
------------------------------

**Comment by agracia-foticos** [comment]
I see this error any time. Remove node_modules, package-lock.json, upgrade all libraries to latest version (vue and nuxt) and works for me

**Reply by stanpodoba** [reply]
With new versions of packages it works! Thank you!

- Nuxt Version:     3.13.1
- CLI Version:      3.13.1
- Nitro Version:    2.9.7
- Vue:                   3.5.4

**Comment by Thanawat1997** [comment]
I have the same problem on docker when I use the `yarn` package manager to install packages.

I solved this problem in 2 ways:

Solution 1. I switch from `yarn` to `npm` to install the packages and other package that same version

Solution 2. I upgrade the `nuxt` and `vue` packages with same package manager
```
  "devDependencies": {
    ...
    "nuxt": "3.13.1",
    "vue": "3.5.4",
  }
```

**Reply by stanpodoba** [reply]
Yeah, upgrade to latest version of Vue resoled the problem. Thank you!
